packages/database-types/index.d.ts

```diff
@@ -68,20 +68,20 @@ export interface Query {
   limitToLast(limit: number): Query;
   off(
     eventType?: EventType,
-    callback?: (a: DataSnapshot, b?: string | null) => any,
-    context?: Object | null
-  ): any;
+    callback?: (a: DataSnapshot, b?: string) => any,
+    context?: Object
+  ): void;
   on(
     eventType: EventType,
-    callback: (a: DataSnapshot | null, b?: string) => any,
-    cancelCallbackOrContext?: Object | null,
-    context?: Object | null
-  ): (a: DataSnapshot | null, b?: string) => any;
+    callback: (a: DataSnapshot, b?: string) => any,
```

Comment:
`b` (aka `prevChildName`) will be null in the case of child_added events where there's no predecessor.